tests/FlatBuffers.Test.Swift/Tests/FlatBuffers.Test.SwiftTests/monster_test_generated.swift

```diff
@@ -1010,6 +1010,7 @@ public struct Monster: FlatBufferObject {
     }
 
     public static func pack(_ builder: inout FlatBufferBuilder, obj: inout MonsterT) -> Offset<UOffset> {
+        let __pos = obj.pos.map { createVec3(x: $0.x, y: $0.y, z: $0.z, test1: $0.test1, test2: $0.test2, test3a: $0.test3.a, test3b: $0.test3.b) }
```

Comment:
What is the minimum swift supported version of the `obj.pos.map` API? Cause if I remember correctly it's not available in 5.0/5.1? correct me if I am wrong. 

because if it's not available on those swift versions we will just break the swift code completely

Reply:
I am not aware of which version. From Apple's doc, it seems available since Swift 2? https://developer.apple.com/documentation/swift/optional/1539476-map

Reply:
It was just weird to see it on an optional! :D but if it has been there since swift 2, then go for it.

Reply:
Yeah, it is super convenient at times when you need some kind but not exactly optional chaining.